c/src/ml-api-service-offloading.h

```diff
@@ -14,17 +14,61 @@
 #define __ML_SERVICE_OFFLOADING_INTERNAL_H__
 
 #include <ml-api-service.h>
+#include <nnstreamer-edge.h>
+
 #include "nnstreamer-tizen-internal.h"
 
 #ifdef __cplusplus
 extern "C" {
 #endif /* __cplusplus */
 typedef void * ml_service_offloading_h;
 
+/**
+ * @brief Enumeration for ml-offloading type.
+ */
+typedef enum
+{
+  ML_OFFLOADING_TYPE_UNKNOWN = 0,
+  ML_OFFLOADING_TYPE_TRAINING,
+
+  ML_OFFLOADING_TYPE_MAX
+} ml_offloading_type_e;
+
+/*/
+ * @brief Structure for ml_service_offloading
+ */
+typedef struct
+{
+  nns_edge_h edge_h;
+  nns_edge_node_type_e node_type;
+
+  gchar *path; /**< A path to save the received model file */
+  ml_option_h info;
+  GHashTable *table;
+
+  ml_offloading_type_e offloading_type;
+  void *priv;
+} _ml_service_offloading_s;
```

Comment:
Why exported?

Reply:
Yes, it will be reflected in the final commit.